README.md

```diff
@@ -1,25 +1,13 @@
-# Rsync - Docker mod for openssh-server
+# Package Install - Universal Docker mod
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+Using this mod you can install any package during starup by providing it through the environment variable `INSTALL_PACKAGES`. This is then passed into the installation command as such: `apt install -y --no-install-recommends ${INSTALL_PACKAGES}` or `apk add --no-cache ${INSTALL_PACKAGES}` for Alpine based images.
 
-In openssh-server docker arguments, set an environment variable `DOCKER_MODS=linuxserver/mods:openssh-server-rsync`
+In any docker container arguments, set an environment variable `DOCKER_MODS=linuxserver/mods:universal-package-install`
 
-If adding multiple mods, enter them in an array separated by `|`, such as `DOCKER_MODS=linuxserver/mods:openssh-server-rsync|linuxserver/mods:openssh-server-mod2`
+If adding multiple mods, enter them in an array separated by `|`, such as `DOCKER_MODS=linuxserver/mods:universal-package-install|linuxserver/mods:universal-stdout-logs`
 
-# Mod creation instructions
-
-* Fork the repo, create a new branch based on the branch `template`.
-* Edit the `Dockerfile` for the mod. `Dockerfile.complex` is only an example and included for reference; it should be deleted when done.
-* Inspect the `root` folder contents. Edit, add and remove as necessary.
-* Edit this readme with pertinent info, delete these instructions.
-* Finally edit the `.github/workflows/BuildImage.yml`. Customize the build branch, and the vars for `BASEIMAGE` and `MODNAME`.
-* Ask the team to create a new branch named `<baseimagename>-<modname>`. Baseimage should be the name of the image the mod will be applied to. The new branch will be based on the `template` branch.
-* Submit PR against the branch created by the team.
-
-
-## Tips and tricks
-
-* To decrease startup times when multiple mods are used, we have consolidated `apt-get update` down to one file. As seen in the [nodejs mod](https://github.com/linuxserver/docker-mods/tree/code-server-nodejs/root/etc/cont-init.d)
-* Some images has helpers built in, these images are currently:
-    * [Openvscode-server](https://github.com/linuxserver/docker-openvscode-server/pull/10/files)
-    * [Code-server](https://github.com/linuxserver/docker-code-server/pull/95)
+For example, to install `rsync`, `git` and `nginx` add the following lines to your docker compose service:
+```yaml
+- DOCKER_MODS=linuxserver/mods:universal-package-install
+- INSTALL_PACKAGES=rsync git nginx
```

Comment:
It's best to have the vars separated by a delimiter like a comma instead of space, because var entry is handled differently between compose and cli

Reply:
I did think about this initially, but this way the env. var can be passed directly into the apk / apt command. This would allow for passing in further command flags without it being messed up by additional parsing that would have to be done.

Applying propper quotes should solve that though. I see that in my example I did not yet include that.

Reply:
> This would allow for passing in further command flags

It's not proper to pass in command flags that way. Please use a delimiter for the package list and you can create a separate var for command flags if need be.
We'd rather not rely on quotes due to them being handled very differently between cli and the various different entry types in yaml.

Reply:
Understandable. Reworked this to use the same delimiting as using multiple mods.